pkg/config/samples/default.dusk.toml

```diff
@@ -34,6 +34,10 @@ streamErrors=true
 # port for the node to bind on
 port=7000
 
+# disable broadcast for topics.Block. 
+# Suitable when broadcasting only in kadcast network
+disableBroadcast=false
```

Comment:
Nitpick: maybe consider calling it `broadcastBlock` so the config parameter is more descriptive, and so that there is no double negative such as in this case `disable...=false`

So the equivalent to what's written here would then be `broadcastBlock=true`

Reply:
Cool, make sense!